src/main/java/Duke.java

```diff
@@ -1,10 +1,33 @@
+import java.util.Scanner;
+
+import functions.AnalyseCommand;
+import functions.TaskList;
+import static functions.DefaultMessages.greet;
+import static functions.DefaultMessages.bye;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String input;
+        Scanner in = new Scanner(System.in);
+        TaskList currentTaskList = new TaskList();
+
+        greet();
+
+        while (true) {
+            input = in.nextLine();
+            AnalyseCommand userCommands = new AnalyseCommand(input);
+            if (userCommands.isBye()) {
+                break;
+            } else if (userCommands.isList()) {
+                currentTaskList.printAllTasks();
+            } else if (userCommands.isDone()) {
+                currentTaskList.setTaskAsDone(userCommands.getCommandDescription());
+            } else {
+                currentTaskList.addTask(userCommands.getCommandType(),userCommands.getCommandDescription(),userCommands.getCommandTime());
```

Comment:
Maybe find a way to shorten this line? It seems too long.

Reply:
Resolved. Thank you!